src/widgets/medications/medications.resource.tsx

```diff
@@ -33,17 +80,31 @@ export function fetchPatientMedications(
   patientID: string,
   status: string = "ACTIVE"
 ): Observable<PatientMedications[]> {
-  return openmrsObservableFetch(
-    `/ws/rest/v1/order?patient=${patientID}&careSetting=${CARE_SETTING}&status=${status}&asOfDate=${dayjs(
-      new Date()
-    ).format(
-      "YYYY-MM-DD"
-    )}&v=custom:(uuid,orderNumber,accessionNumber,patient:ref,action,careSetting:ref,previousOrder:ref,dateActivated,scheduledDate,dateStopped,autoExpireDate,orderType:ref,encounter:ref,orderer:ref,orderReason,orderType,urgency,instructions,commentToFulfiller,drug:(name,strength,concept),dose,doseUnits:ref,frequency:ref,asNeeded,asNeededCondition,quantity,quantityUnits:ref,numRefills,dosingInstructions,duration,durationUnits:ref,route:ref,brandName,dispenseAsWritten)`
+  return openmrsObservableFetch<FetchResponse<PatientMedications>>(
+    `/ws/rest/v1/order?patient=${patientID}&careSetting=${CARE_SETTING}&status=${status}&v=custom:(uuid,orderNumber,accessionNumber,patient:ref,action,careSetting:ref,previousOrder:ref,dateActivated,scheduledDate,dateStopped,autoExpireDate,orderType:ref,encounter:ref,orderer:ref,orderReason,orderType,urgency,instructions,commentToFulfiller,drug:(name,strength,concept),dose,doseUnits:ref,frequency:ref,asNeeded,asNeededCondition,quantity,quantityUnits:ref,numRefills,dosingInstructions,duration,durationUnits:ref,route:ref,brandName,dispenseAsWritten)`
   ).pipe(
     map(({ data }) => {
-      const meds = [];
+      const meds: Array<PatientMedications> = [];
       data["results"].map(result => {
-        if (result.orderType.name === "Drug Order") {
+        if (result.orderType.display === "Drug Order") {
```

Comment:
Can we put this in a constant somewhere? It seems so arbitrary that it may change? Not sure here. Is this not localized etc. from the API? Are we sure its always "Drug Order"?

Reply:
It is a order type [Drug Order]( https://wiki.openmrs.org/display/docs/Order+Entry+API?lng=es#OrderEntryAPI-PlaceADrugOrder.1)

Reply:
In the referenced document its:

```ts
"type" : "drugorder",
```

But then again - put it somewhere in a constant. We use this magic string quite some times and I would love to have some easy of maintenance and reusability.